agar-io/cigar/Dockerfile

```diff
@@ -1,19 +1,19 @@
-FROM alpine/git:v2.26.2 AS clone
+FROM alpine/git:v2.32.0 AS clone
 RUN git config --global advice.detachedHead false
 RUN git clone https://github.com/opcon/Cigar2.git /app
 WORKDIR /app
 RUN git checkout ced18851717d2b18eef9e8e367e35cf399c50d6c
 RUN rm -rf .git
 
 
-FROM node:8.17.0-alpine3.11
+FROM node:16.13.0-alpine
 EXPOSE 3000
 ENV NPM_CONFIG_LOGLEVEL info
 WORKDIR /app
 COPY --from=clone /app/app/package*.json /app/
-RUN yarn install
+RUN yarn install --ignore-engines
```

Comment:
What does this option do?

Reply:
cigar [hardcodes](https://github.com/opcon/Cigar2/blob/master/app/package.json#L13) the long-deprecated `"node": "8.x"`. This overrides the version check during `yarn install`, which will otherwise refuse to run.